p2p/service/sim.go

```diff
@@ -252,9 +303,9 @@ func (sn *Node) Update(node2 node.Node) {
 // Shutdown closes all node channels are remove it from the Simulator map
 func (sn *Node) Shutdown() {
 	sn.sim.mutex.Lock()
-	for _, c := range sn.sim.protocolHandler[sn.Node.PublicKey().String()] {
+	for _, c := range sn.sim.protocolDirectHandler[sn.Node.PublicKey().String()] {
 		close(c)
 	}
-	delete(sn.sim.protocolHandler, sn.Node.PublicKey().String())
+	delete(sn.sim.protocolDirectHandler, sn.Node.PublicKey().String())
```

Comment:
should be doing the same for protocolGossipHandler now..

Reply:
done